webapp/public/app.css

```diff
@@ -77,6 +77,10 @@ body {
   display: none;
 }
 
+.board [type="submit"].show {
```

Comment:
What is the purpose ? Every tile in board are [type="submit"] and should be hidden

Reply:
I use a submit button for the "start new game" page.